src/spyglass/spikesorting/spikesorting_sorting.py

```diff
@@ -202,12 +202,10 @@ def make(self, key: dict):
                 sampling_frequency=recording.get_sampling_frequency(),
             )
         else:
-            # whiten recording; make sure dtype is float16
-            recording = sip.whiten(recording, dtype="float16")
-            if sorter_params["whiten"] == True:
-                print(
-                    "Warning: the recording is whitened prior to sorting but the sorter param includes whitening"
-                )
+            # whiten recording here if indicated, so can make sure dtype is float16
+            if sorter_params["whiten"]:
```

Comment:
With this logic, if `sorter_params["whiten"] = False`, then the recording would not be whitened. But I think we always want to whiten the recording before sorting. How about we whiten first (`sip.whiten(recording, dtype="float16")`) and then set `sorter_params["whiten"] = False` if sorter is MS4?

Reply:
Thanks @khl02007. If we always want to whiten, then it seems like it would be best to not have a parameter for whitening at all, and just make sure this code is backwards compatible (if people do pass in a parameter since it used to be one, raise an error if the parameter is set to False, and remove the parameter if True). What are your thoughts on this?